### ISSUE TITLE
Torn (557520)

### ISSUE THREAD

**Issue by farmboy0** [ValveSoftware/Proton#5425]
# Compatibility Report
- Name of the game with compatibility issues: Torn
- Steam AppID of the game: 557520

## System Information
- GPU: RX5700XT
- Driver/LLVM version: Mesa 21.3.2 LLVM13
- Kernel version: 5.15
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/farmboy0/16f79f8c08e6062bb1cf5365963499ce
- Proton version: 6.3-8c

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-557520.log.gz](https://github.com/ValveSoftware/Proton/files/7745779/steam-557520.log.gz)

## Symptoms 
The game crashes after about 15-30 minutes playtime.
From the kernel log:
[26722.344944] VKRenderThread[25645]: segfault at 7efbfc385918 ip 00007f142656ded8 sp 00007f13c373ad90 error 6 in vrclient.so[7f1426336000+6ac000]
[26722.344962] Code: dc 49 00 e8 7a 06 e5 ff 48 8b 98 18 00 00 00 49 8d 44 24 20 48 89 43 10 49 8b 54 24 20 66 0f ef c0 49 8b 44 24 18 48 83 e2 fe <48> 89 42 f8 49 8b 54 24 20 48 83 e0 fe 48 89 10 31 c0 41 0f 11 44

## Reproduction
Play the game for some time

**Comment by kisak-valve** [comment]
Hello @farmboy0, these look like a couple lines of interest from your log:
```
err:   27555.548:0140:0144:trace:vrclient:winIVRSystem_IVRSystem_012_GetEyeToHeadTransform 0x132e2310
DXVK: Failed to create GPU event
```
This is followed by an access violation (c0000005) and the log ends shortly after that.

**Comment by farmboy0** [comment]
@kisak-valve Yes this is probably where the crash in vrclient.so happens, but winIVRSystem_IVRSystem_012_GetEyeToHeadTransform is called before the crash already.

**Comment by pwaller** [comment]
Judging by the `code` it looks like Google Earth VR ([comment](https://github.com/ValveSoftware/Proton/issues/3408#issuecomment-1381994773)) also segfaults in the same place after some time inside the app.
